device/rsvp/rsvpEro.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    Rsvp.Ero:
+      description: >-
+        Configuration for the optional RSVP-TE ERO object included in Path Messages. 
+      type: object
+      properties:
+        prepend_egress_ip:
+          description: >-
+            Determines whether the IP address of the egress endpoint of the LSP should be added as an ERO sub-object.
+            If it is to be included, it can be included as a Loose hop or as a Strict hop.
+          type: string
+          enum:
+          - dont_prepend
+          - prepend_loose
+          - prepend_strict
+          default: prepend_loose
+        prefix_length:
+          description: >-
+            If prepend_egress_ip is set to one of 'prepend_loose' or 'prepend_strict', then set this value as the prefix length of the ERO sub-object containing egress IP address.                        
+          type: integer
+          default: 32
+          minimum: 0
```

Comment:
Curious what does prefix length 0 will do ?

Reply:
It will probably use the default route to forward the Path message. Checked that RFC does not disallow length 0.